web/themes/contrib/civictheme/includes/search.inc

```diff
@@ -28,6 +28,15 @@ function civictheme_preprocess_block__civictheme_search(array &$variables): void
   if ($link !== NULL) {
     $variables['url'] = $link->getUrl()->toString();
     $variables['text'] = $link->get('title')->getString();
+    if ($entity->hasField('field_c_b_link_in_mobile_menu') && civictheme_get_field_value($entity, 'field_c_b_link_in_mobile_menu')) {
```

Comment:
`civictheme_get_field_value` runs a has-field check within, so this could just be simplified to
`if (civictheme_get_field_value($entity, 'field_c_b_link_in_mobile_menu')) {`

Reply:
@febdao Please change to @alan-cole suggestion